src/main/java/uk/gov/pay/directdebit/payments/api/PaymentRequestResponse.java

```diff
@@ -15,11 +13,11 @@
 @JsonFormat(shape = JsonFormat.Shape.OBJECT)
 public class PaymentRequestResponse {
     @JsonProperty("links")
-    private List<Map<String, Object>> dataLinks = new ArrayList<>();
+    private List<Map<String, Object>> dataLinks;
```

Comment:
Considering it again, it might be better to have a default of empty list. That's what we do for cards and we might want to avoid having to do different things in publicapi depending on the connector. Nicely spotted.